demo/rust/filedemo/Cargo.toml

```diff
@@ -1,11 +1,11 @@
 [package]
 name = "pv_rhino_filedemo"
-version = "0.1.0"
```

Comment:
was it that old?! :)

Reply:
No I think we just never bothered to update the demo versions in the past.